mmv1/third_party/terraform/services/bigquery/resource_bigquery_table_test.go

```diff
@@ -1655,9 +1655,9 @@ resource "google_bigquery_table" "test" {
   time_partitioning {
     type                     = "%s"
     field                    = "ts"
-    require_partition_filter = true
     expiration_ms            = 1000
   }
+  require_partition_filter = true
```

Comment:
Can you please add a new test to leave `require_partition_filter` inside `time_partitioning`? Or keep one test that `require_partition_filter` is inside `time_partitioning`?

Reply:
Done